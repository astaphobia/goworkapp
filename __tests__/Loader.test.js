import React from 'react'
import { shallow } from 'enzyme'

import Loader from '@/components/Loader'

describe('Loader', () => {
    it('hope doing well Loader', () => {
        const comp = shallow(<Loader />)
        expect(comp).toMatchSnapshot()
    })
})