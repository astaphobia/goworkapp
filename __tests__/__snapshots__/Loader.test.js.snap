// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader hope doing well Loader 1`] = `
<div
  className="centered text-center"
  style={
    Object {
      "top": "200px",
    }
  }
>
  <svg
    className="spinner"
    height="65px"
    viewBox="0 0 66 66"
    width="65px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      className="path"
      cx="33"
      cy="33"
      fill="none"
      r="30"
      strokeLinecap="round"
      strokeWidth="6"
    />
  </svg>
</div>
`;
